extensions/amp-story/1.0/amp-story.js

```diff
@@ -1651,6 +1655,19 @@ export class AmpStory extends AMP.BaseElement {
   triggerActiveEventForPage_() {
     // TODO(alanorozco): pass event priority once amphtml-story repo is merged
     // with upstream.
+    if (this.viewerMessagingHandler_) {
```

Comment:
```suggestion
    if (this.viewerMessagingHandler_ && this.activePage_) {
```

Reply:
Is this needed? Just wondering since we are writing this in the function which is called just after initializing the active page.

Reply:
Yes, since `this.activePage_` is initialized as `null`, the type checking will complain since it doesn't know that is has been given a `non-null` value at this point of the codepath.

Reply:
Cool, got it, thanks for the explanation.